libs/application/templates/financial-aid/src/fields/Status/ApplicantStatus.tsx

```diff
@@ -42,8 +43,11 @@ const ApplicantStatus = ({ application }: FAFieldBaseProps) => {
         />
       )}
 
-      {/* TODO: redirect user to page to upload files when button is clicked insied of MissingFilesCard*/}
-      {state === ApplicationState.DATANEEDED && <MissingFilesCard />}
+      {state === ApplicationState.DATANEEDED && (
+        <MissingFilesCard
+          onClick={() => goToScreen && goToScreen('missingFiles')}
```

Comment:
why have this onclick even, and just have it  goToScreen?: (id: string) => void prop